src/components/tables/TableConnected.tsx

```diff
@@ -57,16 +59,46 @@ export const getTableCompositeState = (state: IReactVaporState, id: string): ITa
   } as ITableCompositeState;
 };
 
+const getDataById = (data: ITableData, props: ITableOwnProps): ITableById => data.byId;
+
+const getSelectedIds = (data: ITableData, props: ITableOwnProps): string[] => data.selectedIds || [];
+
+const getActions = (data: ITableData, props: ITableOwnProps): (rowData?: IData) => IActionOptions[] => props.getActions;
```

Comment:
consider typing (rowData?: IData) => IActionOptions[] as `IGetActions` here i think it would be more readable